### ISSUE TITLE
Strange behiavour

### ISSUE THREAD

**Issue by boruok** [rxi/console#4]
**lite version:**
1.05
**OS/device including version:**
Windows 10
**Issue description:**
Can't run console build-command.
**Steps to reproduce:**
1. copy lite archive
2. extract to C:/lite
3. Navigate to the data/plugins folder and run the following command: git clone https://github.com/rxi/console
4. navigate to project root folder and drag and drop into lite.exe or lite windows itself
5. in lite press "ctrl+p" -> core: open project module
6. copy+paste template below
8. press "ctrl+s" to save file
9. close editor and open again
10. press "ctrl+b" to run build command
11. **fail**

I've noticed strange thing with command palette (ctrl+shift+p) looks like drag and drop doesn't load project module into editor, you can check this with command palette list:
1. (if editor is preffered for current extension) open any file in root folder of project: project: build-project appears in list
2. drag and drop into lite.exe: -> not
3. drag and drop into window -> not

here is my config:
```lua
local core = require "core"
local command = require "core.command"
local keymap = require "core.keymap"
local console = require "plugins.console"

command.add(nil, {
  ["project:build-project"] = function()
    core.log "Building..."
    console.run {
      command = "godot",
      file_pattern = "(.*):(%d+):(%d+): (.*)$",
      on_complete = function() core.log "Build complete" end,
    }
  end
})

keymap.add { ["ctrl+b"] = "project:build-project" }
```

**Comment by rxi** [comment]
I haven't gotten round to trying to reproduce the problem in the first part of your comment -- it may have been helpful if step 10 provided a little more detail beyond just "fail"!

Regarding the second part: this is a problem in core which I plan to fix before the next release, thanks for bringing it to my attention.

**Comment by boruok** [comment]
oh, my bad. **10. fail** is also doesn't executes (doesn't is added into command palette)

**Comment by boruok** [comment]
fixed with latest **1.06** update.
